FileWritingObjects/FileWritingObjects/SetWriter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+
+namespace FileWritingObjects
+{
+    class SetWriter : IDisposable
+    {
+        private bool DisposedValue = false;
+        private StreamWriter? Writer;
```

Comment:
```suggestion
        private StreamWriter _Writer;
```
You don't need to mark this as nullable since it gets set in your constructor.